src/components/Search/SearchDropdown.tsx

```diff
@@ -15,7 +15,9 @@ function SearchDropdown(props: SearchDropdownProps) {
 
   return (
     <FormControl sx={{ width: '100%' }} size={size}>
-      <InputLabel id="select-repository">repository</InputLabel>
+      <InputLabel id="select-repository">
+        repository
+      </InputLabel>
```

Comment:
Is this change needed?

Reply:
@jrbenny35 It was not but I just thought it would follow good formatting. I will change it back